### ISSUE TITLE
"unable to connect to source LXD" is back

### ISSUE THREAD

**Issue by ddennedy** [samuelmeuli/action-snapcraft#29]
I see this has been reported before, but I opened this anew because the situation is different at least time-wise. This just started happening again a day or two ago. Hard to be exact because my nightly builds started failing a couple of days ago for a different reason. 
See https://github.com/mltframework/shotcut/runs/2079434751?check_suite_focus=true

I noticed that this action was recently released as v1.2.0, and I upgraded to that but it did not help. I have very limited experience with lxd/lxc. I guess lxd is not running.

**Comment by casperdcl** [comment]
looks like a duplicate of #24/#23

From https://github.com/samuelmeuli/action-snapcraft#basic you need `runs-on: ubuntu-18.04`

**Comment by ddennedy** [comment]
Thank you. I saw #23 previously, but without digging into the details it just looked like there was a fix that was applied 3 months ago. It was not obvious that `ubuntu-20.04` is not supported. Yes, the readme specifically mentions 18.04, which is helpful, but not clear that 20.04 does not work. One might easily assume a Ubuntu-based technology like snap would work on `ubuntu-latest`, which is what we were using. I have changed my workflow to 18.04, and now it works.
